src/toil/jobStores/aws/jobStore.py

```diff
@@ -1161,11 +1161,11 @@ def writeTo(self, writable):
                     elif info.version:
                         headers = info._s3EncryptionHeaders()
                         key = info.outer.filesBucket.get_key(bytes(info.fileID), validate=False)
-                        for attempt in retry_s3():
-                            with attempt:
-                                key.get_contents_to_file(writable,
-                                                         headers=headers,
-                                                         version_id=info.version)
+                        with open(writable, 'wb') as fileObject:
```

Comment:
I don't think this is going to work. Writable is already a file object; you probably can't `open()` it again.